### ISSUE TITLE
Unknown variable: 'loki::service::service_notify

### ISSUE THREAD

**Issue by saz** [attachmentgenie/attachmentgenie-loki#32]
I see the following error during a puppet run (introduced with v1.1.0).

This looks like a scope issue to me, but maybe I'm missing something here

```
Error: Could not retrieve catalog from remote server: Error 500 on SERVER: Server Error: Evaluation Error: Unknown variable: 'loki::service::service_notify'. (file: /etc/puppetlabs/code/environments/production/modules/loki/manifests/config.pp, line: 22, column: 15) on node example.com
```

**Comment by attachmentgenie** [comment]
It does smell like it doesn't it, lets fire up some test nodes

**Comment by trenta** [comment]
On a fresh install this also occurs in install.pp
Error: Could not retrieve catalog from remote server: Error 500 on SERVER: Server Error: Evaluation Error: Unknown variable: 'loki::service::service_notify'. (file: /etc/puppetlabs/code/environments/production/modules/loki/manifests/config.pp, line: 22, column: 15)

**Comment by saz** [comment]
I've investigated this issue a bit more. 
In https://github.com/attachmentgenie/attachmentgenie-loki/blob/0cf4621fcbf8a54f411f314338175ecc7c047480/manifests/config.pp#L22 the variable `$loki::service::service_notify` is used, but it will be defined in https://github.com/attachmentgenie/attachmentgenie-loki/blob/0cf4621fcbf8a54f411f314338175ecc7c047480/manifests/service.pp#L38.

If we're looking at `init.pp`:
```
  anchor { 'loki::begin': }
-> class{ '::loki::install': }
-> class{ '::loki::config': }
~> class{ '::loki::service': }
-> anchor { 'loki::end': }
```
We can see, that `loki::config` will be included *before* `loki::service`.

I'll send in a PR asap

**Comment by attachmentgenie** [comment]
i just released v2.0.0
